SavedInstances/SavedInstances.lua

```diff
@@ -4580,3 +4591,16 @@ StaticPopupDialogs["SAVEDINSTANCES_DELETE_CHARACTER"] = {
   enterClicksFirstButton = false,
   showAlert = true,
 }
+
+StaticPopupDialogs["SAVEDINSTANCES_REPORT_KEYS"] = {
+  preferredIndex = 3, -- reduce the chance of UI taint
```

Comment:
```suggestion
  preferredIndex = STATICPOPUPS_NUMDIALOGS, -- reduce the chance of UI taint
```

Reply:
FYI, this is straight up c&p from the dialog above (`SAVEDINSTANCES_DELETE_CHARACTER`). Should I change this there too while I'm at it?

Reply:
Haha yeah, sorry.

Reply:
Travis is complaining about undefined variable, should I just define this? Does Lua distinguish between variables and constants? Is this defined somewhere else?

Reply:
Ah yup, `STATICPOPUPS_NUMDIALOGS` is missing in the luacheckrc